client/packages/common/src/utils/arrays/SortUtils.ts

```diff
@@ -26,8 +26,13 @@ const parseValue = <T, K extends keyof T>(object: T, key: K) => {
 };
 
 const getSortByExpiry =
-  (desc = false) =>
+  (desc = false, nonExpiringLast = false) =>
```

Comment:
Old way was leaving the lines with no expiry value in place? Probably mostly a dev data problem, but I would think "non-expiring" batches should be allocated last

Reply:
> "non-expiring" batches should be allocated last

Yep that's how it's done on b/e